pom.xml

```diff
@@ -45,10 +45,10 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <java.version>1.6</java.version>
-    <project.google.http.version>1.19.0</project.google.http.version>
+    <project.google.http.version>1.23.0</project.google.http.version>
     <project.junit.version>4.12</project.junit.version>
-    <project.guava.version>19.0</project.guava.version>
-    <project.appengine.version>1.9.34</project.appengine.version>
+    <project.guava.version>20.0</project.guava.version>
```

Comment:
Likely. However other projects upstream of this including google-cloud-java depend on Guava 20 and are already pulling that version in instead of 19.0